src/cloudlabs/deployer/terraform/terraform.tf_template

```diff
@@ -125,7 +125,7 @@ resource "azurerm_virtual_machine" "vm" {
     }
 
     os_profile {
-        computer_name = "{{host.dns_name}}"
+        computer_name = "cloudlabsvm"
```

Comment:
This doesn't have to match the vm resource name, right? Otherwise we should probably have it reference that, rather than be hardcoded.